lib/describe-metadata-service.js

```diff
@@ -41,7 +41,8 @@ var childTypes = [{
 	xmlName: 'CompactLayout',
 	tagName: 'compactLayouts',
 	parent: 'CustomObject',
-	key: 'fullName'
+	key: 'fullName',
+	isNamed: true
```

Comment:
This is the compact layout fix

Reply:
@froucher Could you please create a separate PR for the CompactLayout fix? Then we can merge it right away.

Reply:
This is the new PR #207 

Thanks!

Reply:
Added SharingReason and BusinessProcess similar fix to the PR #232